lstchain/image/muon/muon_analysis.py

```diff
@@ -125,13 +127,34 @@ def fit_muon(x, y, image, geom, tailcuts):
     image_clean: `np.ndarray`
         Image after cleaning
     """
-
+    
+    if tailcuts == None:
+        #we want to quantize the noise of the image. To do so, we will use the negative Q cumulative distribution.
+        negative_Q = np.sort(image[image <= 0])
+        
+        hist, bins = np.histogram(negative_Q,range=(-15,0),bins=30)
+        bins=bins[:-1]
+        
+        normalised_cumulative_hist = np.asarray(np.cumsum(hist)/np.sum(hist))
+
+        idx = (np.abs(normalised_cumulative_hist - 0.318)).argmin() #Find q closest to standard deviation
+        dev = np.abs(bins[idx])
+        #MORALEJO: we want to get, from a single image, a quantity related to the width of the noise distribution, 
```

Comment:
this comment can probably be edited rather than copy/paste that of:
https://github.com/cta-observatory/cta-lstchain/pull/1012#discussion_r964953348